RationaleMCP/0036/README.md

```diff
@@ -39,19 +53,46 @@ One goal of "Clocked Discretized Continuous-Time Partition" is to preserve the b
 In addition the current reinit might possibly in the future be replaced by some form of impulses, indicating that the changes are physical and influence other parts of the system, this does not seem consistent with how states are set in this MCP.
 Finally having a conditional setting of states is more complicated, and reinit and other similar operators naturally suggest that - and we either need to determine what it means or forbid it.
 
+## Proposed alternative
+Use a special binding equation:
+
+Real x=measurement(xs);
+
+This special operator, measurement, is only legal in a "Clocked Discretized Continuous-Time Partition" and only in this way and means that 'x' is first seen as a state, but after index reduction x is no longer a state and x=xs replaces the integration of der(x).
+
+The name of the operator can be discussed. The prototype re-used "reinit" to avoid introducing new reserved words.
```

Comment:
Regarding the name of the operator, I'd prefer `observation` over `measurement`, as I think that `xs` will in general be computed using an observer.  At least this is the terminology that I'm familiar with from the field of automatic control.

In some more detail: Only in idealized situations like when the plant happens to be a completely transparent Modelica model, will there be measurements that are the direct (noise-free) measurements of the states of interest — otherwise, measurements will be fed to the observer, which is then responsible for computing estimates of the states of interest.

Reply:
> Regarding the name of the operator, I'd prefer `observation` over `measurement`, as I think that `xs` will in general be computed using an observer. At least this is the terminology that I'm familiar with from the field of automatic control.

I agree that "measurement" can give the wrong idea, but I'm not sure if "observation" is better; to me they sound somewhat synonymous (except that "observation" sounds more impressive) and as the the input to the observer component - not its output.

What about "estimate"?

Reply:
Yes, that's better.

A related question is if there would be something that could alleviate the inherent confusion of the roles of the two variables in `x = estimate(xs)` — who wouldn't read this as _x is an estimate of xs_ (rather than the intended _x is set to the estimate xs_)?